### DISCUSSION TITLE
Confetti on reaching a new PB

### DISCUSSION THREAD

**Discussion by unrealapex** [monkeytypegame/monkeytype#1859] [Ideas] [answered]
It would be cool if Monkeytype could show some confetti when you reach a new personal best.

**Comment by Miodec** [comment]
Not sure it fits the minimalistic design.

**Reply by unrealapex** [reply]
Maybe having it as an easter egg? Like have it only happen if you've increased by a certain wpm.

**Reply by frogfrogfrogfrogfrog** [reply]
It could work as something else. For example in "power mode" instead of the explody thingies you could have confetti pop out everytime you type a letter. You could also have it as a funbox where your vision is clouded by hordes of confetti.

**Reply by abd11a** [reply]
like this! https://ash-k.itch.io/textreme
sorta anti-minimalist though.

**Reply by unrealapex** [reply]
Thinking more of [this](https://github.com/catdad/canvas-confetti).

**Comment by unrealapex** [comment, marked as answer]
Feature got added.
